src/main/play-doc/developer/Control-API.md

```diff
@@ -0,0 +1,78 @@
+# Typesafe ConductR %PLAY_VERSION%
+
+
+## The Control API
+
+ConductR's control protocol is RESTful and has the following functional scope:
+
+* load a bundle and optionally its configuration
+* scale a bundle i.e. starting it and possibly stopping it
+* unload a bundle
+* query bundle state
+* query member state
+* subscribe to Server Sent Events (SSEs) regarding state changes to bundles and members
+
+## Load a bundle
+
+Upload a bundle and optionally, its configuration to ConductR. 
+
+Here is an example of uploading a Visualizer bundle without any configuration. [Curl](http://curl.haxx.se/) is being used.
+
+```
+curl \
+  --form system=sys \
+  --form bundleName=http-server \
+  --form nrOfCpus=2 \
+  --form memory=104857600 \
+  --form diskSpace=104857600 \
+  --form roles=web-server \
+  --form bundle=@visualizer/target/bundle/visualizer-0.1.0-cfe2a36795fd78507c4d2b5817152ae449e4acd9d5ea94d1f604d2c11417e40f.zip \
+  http://localhost:9005/bundles
+```
+
+### Request
+
+```
+POST /bundles
+```
+
+The following fields are provided as multipart/form-data fields:
+
+Field            | Description
+-----------------|------------
+system           | As per its equivalent property in [bundle.conf](Bundles.html)
+bundleName       | As per its equivalent property in [bundle.conf](Bundles.html)
+nrOfCpus         | As per its equivalent property in [bundle.conf](Bundles.html)
+memory           | As per its equivalent property in [bundle.conf](Bundles.html)
+roles            | As per its equivalent property in [bundle.conf](Bundles.html)
+bundle           | The file that is the bundle. The filename is important with its hex digest string and is required to be consistent with the SHA-256 hash of the bundle's contents. Any inconsistency between the hashes will result in the load being rejected.
+configuration    | Optional. Similar in form to the bundle, only that is the file that describes the configuration. Again any inconsistency between the hex digest string in the filename, and the SHA-256 digest of the actual contents will result in the load being rejected.
+
+### Responses
+
+##### Success
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "requestId": "{request-id}",
+  "bundleId": "{bundleId}"
+}
+```
+
+Field        | Description
+-------------|------------
+requestId    | A UUID for this request.
+bundleId     | An identifier directly related to the digest of a bundle and, if supplied, the digest of its configuration. This identifier is guaranteed to remain related to both the bundle and its optional configuration. It is used for subsequent distinctive references by the API.
```

Comment:
Is it possible to link to somewhere where we explain what the bundleId format looks like?